src/dotnet/APIView/ClientSPA/src/environments/environment.ts

```diff
@@ -0,0 +1,6 @@
+export const environment = {
+    production: true,
+    apiUrl: 'https://apiviewuxtest.com/api/',
```

Comment:
I don't know much about angular. How are we going to handle different environments?

Reply:
Oh, Yh, I have added more configurations to accommodate staging and prod-staging deployments.